content/en/Features/multi-region/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Multi-Region"
+linkTitle: "Multi-Region"
+weight: 5
+description: How to use Opta to deploy multi-region environments
+---
+
+## Warning: Here be Dragons!
+Out of the love and responsibility we feel towards our users we must seriously ask you and your team: are you **SURE**
+you need this?
+
+Multi-region, even in a perfect devops world (which opta, nor any tool, is) is a really hard 
+application side problem (i.e. your server code). The users will need to make their code knowingly switch from read to 
+write replicas of the datastore, be tolerant to read lag, ideally even know what region it’s in and keep a sort of 
+network map to know what app is in which region to optimize its http requests, and even more... There’s a ton of ways to 
+mess this up, and it should only be really attempted to reach disparate user locales or maybe for some reliability scenarios 
```

Comment:
it's also much most costly - our users seem to be cost conscious so they should be aware of it.